openfl/component/aggregator/straggler_handling.py

```diff
@@ -2,27 +2,77 @@
 # SPDX-License-Identifier: Apache-2.0
 
 
-"""Cutoff time based Straggler Handling function."""
+"""Straggler handling module."""
+
 import threading
 import time
+from abc import ABC, abstractmethod
 from logging import getLogger
 from typing import Callable
 
 import numpy as np
 
-from openfl.component.straggler_handling_functions.straggler_handling_function import (
-    StragglerHandlingPolicy,
-)
 
+class StragglerHandlingPolicy(ABC):
+    """Federated Learning straggler handling interface."""
+
+    @abstractmethod
+    def start_policy(self, **kwargs) -> None:
+        """
+        Start straggler handling policy for collaborator for a particular round.
+        NOTE: Refer CutoffPolicy for reference.
+
+        Args:
+            **kwargs
+
+        Returns:
+            None
+        """
+        raise NotImplementedError
+
+    @abstractmethod
+    def reset_policy_for_round(self) -> None:
+        """
+        Reset policy variable for the next round.
+
+        Args:
+            None
+
+        Returns:
+            None
```

Comment:
If this is the case for subclassed policies, please eliminate empty lines. It suffices to say `Resets policy for the next round` and not using 6 lines to say it takes or returns nothing. The `-> None` and no args make it obvious to the user in an IDE.

Also on the info itself: It is better to mention what exactly is being reset without leaking into the detail. Example `Resets countdown timer for collaborator reporting until next round`, or along those lines.

Reply:
Accepted. Will incorpoate